src/platform/site-wide/user-nav/components/DropDownPanel/DropDownPanel.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: DropDownPanel
+name: DropDownPanel
+tags: panel, component
+---
+
+import DropDownPanel from './DropDownPanel'
+
+### Code:
+```javascript
+import DropDownPanel from '@department-of-veterans-affairs/component-library/DropDownPanel'
```

Comment:
Should this be the local import from line 7?

Reply:
I think this file is structured (imports) and formatted (backticks) appropriately.

My question, though, is what it's used for. Is this still needed as it's ported over to vets-website? What purpose is it serving? I can't quite figure out what purpose these mdx files are serving even while the files live in the `component-library` repo. They seem to be a relic of a former way of documenting these components, pre Storybook. I cannot say for sure, though. I'd love some insight so I can feel better about whether we need this file in this PR.

Reply:
Yeah, you're right. This file was used by Storybook and is no longer needed